### ISSUE TITLE
Search MergeRequests by iids

### ISSUE THREAD

**Issue by Xiot** [jdalrymple/gitbeaker#3283]
**Description**
There doesn't seem to be a way to get a list of merge requests by their iids.

The Gitlab [documentation](https://docs.gitlab.com/ee/api/merge_requests.html#list-project-merge-requests) shows the `iids[]` query parameter that can be used for this.
```
GET /projects/:id/merge_requests?iids[]=42&iids[]=43
```

**Proposal**

Add the `iids` option to `AllMergeRequestsOptions` and add the `iids[]` query parameters to the request

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
It should work rn, if you pass it as the options:

```
{
iids: [3,3,4]
}
```

But id have to still update the typing

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`38.10.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/38.10.0) :rocket:
